### ISSUE TITLE
Use em for font sizes

### ISSUE THREAD

**Issue by nlhkabu** [pypa/warehouse#6349]
All text should be resizable. 

Note: There are a couple of places in the UI where we use 14px text size - namely form help text and some tables.

This should be reviewed.

**Comment by nlhkabu** [comment]
Note: this is controlled in the SCSS by the variable `$small-font-size`

**Comment by nlhkabu** [comment]
> The only requirement is that you leave the user with the ability to increase the size by 200%. Best practice is to specify the font-size using em so that the font size can be adjusted independently of browser magnification.

https://webaim.org/discussion/mail_thread?thread=7051

**Comment by nlhkabu** [comment]
Problem: Changing the 'font size' in Google chrome settings does not affect the fonts for the site.

**Comment by nlhkabu** [comment]
Line height should also be set in em, so that the height of elements adjusts with the increased font size setting
